### DISCUSSION TITLE
布隆过滤器索引、zoneMap索引、bitmap索引，为什么搞一个二次索引，这个ordina具体是怎么使用的

### DISCUSSION THREAD

**Discussion by easymanlo** [apache/doris#6801] [A - General / Q&A]
![索引文件](https://user-images.githubusercontent.com/48037566/136649426-d083a2e2-7e75-4da7-99a8-4e62afc03a7b.jpg)

**Comment by morningman** [comment]
ordinal index 可以理解为是为了统一索引的表示，所有类型的索引，最终都是通过ordinal index来找到 datapage的
